src/dataflow/src/compute/compute_state.rs

```diff
@@ -363,7 +366,21 @@ impl<'a, A: Allocate, B: ComputeReplay> ActiveComputeState<'a, A, B> {
                 start_offset,
                 self.timely_worker.index(),
                 move |time, data| {
-                    m_logger.publish_batch(time, data);
+                    c_logger.publish_batch(time, data);
+                    activator.activate();
+                },
+            ),
+        );
+
+        let activator = m_activator.clone();
```

Comment:
might want to have a `TODO` here that explains the current scenario about logging not being setup in STORAGE

Reply:
I'll file an issue; I think it's bigger than a TODO.